src/Admin/PairedBrowsing.php

```diff
@@ -73,6 +74,17 @@ public static function is_needed() {
 		);
 	}
 
+	/**
+	 * Get the list of service IDs required for this service to be registered.
+	 *
+	 * @return array<string> List of required services.
```

Comment:
This means the same thing, but any reason for/against?

```suggestion
	 * @return string[] List of required services.
```

Reply:
Either works, just a little copypasta 😄.

Reply:
I'll commit suggestion since it's what's more prominent in the codebase.